infra/terraform/environments/dev/main.tf

```diff
@@ -249,7 +249,126 @@ module "service" {
       {
         name     = "process-queue",
         commands = ["queue:process-queue"],
-        memory   = 2048,
+      },
+      {
+        name     = "ch-vs-olcs-diffs",
+        commands = ["batch:ch-vs-olcs-diffs"],
+      },
+      {
+        name     = "clean-up-variations",
+        commands = ["batch:clean-up-variations"],
+      },
+      {
+        name     = "cns",
+        commands = ["batch:cns"],
+      },
+      {
+        name     = "create-psv-licence-surrender-task",
+        commands = ["batch:create-psv-licence-surrender-task"],
+      },
+      {
+        name     = "psv-operator-list-export",
+        commands = ["batch:data-gov-uk-export -v --report-name=psv-operator-list --path=/tmp/"],
```

Comment:
This may be a little ambiguous at first glance, it may be better to split it into a list of "parts", or change the data type to a `string` instead of a `list`. Though, I'd recommend keeping it as a list, and passing the "parts".

An array of parts executes without a shell, and a string command passes through a shell. In this instance, although this is a single string, it'll be split up anyway and executed without a shell - which may cause ambiguity.